tethys_gizmos/gizmo_options/map_view.py

```diff
@@ -266,16 +329,27 @@ def __init__(self, height='100%', width='100%', basemap='OpenStreetMap', view={'
         self.disable_basemap = disable_basemap
         self.feature_selection = feature_selection
 
-    @staticmethod
-    def get_vendor_js():
+    @classmethod
+    def static_url(cls):
```

Comment:
As we discussed, let’s handle the in between cases CDN >= 5.0.0 in case they support those versions in the future.